components/cmp/ui/src/CmpUi/index.js

```diff
@@ -0,0 +1,131 @@
+import React, {useState, useRef, Suspense} from 'react'
+import PropTypes from 'prop-types'
+import {useMount} from '@schibstedspain/sui-react-hooks'
+import {UI_TYPES} from '../settings'
+
+const CONSENT_STATUS_NOT_ACCEPTED = 'NOT_ACCEPTED'
+const NO_OP = () => {}
+
+const CmpUiBanner = React.lazy(() => import('../CmpUiBanner'))
+const CmpUiModal = React.lazy(() => import('../CmpUiModal'))
+const CmpModalConsents = React.lazy(() =>
+  import('@schibstedspain/react-cmp-modal')
+)
+
+export default function CmpUiContainer({
+  companyName,
+  getConsentStatus,
+  getPurposesAndVendors,
+  lang,
+  logo,
+  onAccept = NO_OP,
+  onConfigure = NO_OP,
+  privacyUrl,
+  sendConsents,
+  type
+}) {
+  const [showModal, setShowModal] = useState(false)
+  const [showNotification, setShowNotification] = useState(false)
+  const [showPartners, setShowPartners] = useState(false)
+  const containerDOMEl = useRef()
+
+  const _handleAccept = async () => {
+    const {
+      purposeConsents,
+      vendorConsents
+    } = await getPurposesAndVendors.execute()
+    await sendConsents.execute({purposeConsents, vendorConsents})
+
+    setShowModal(false)
+    setShowNotification(false)
+
+    _removeBodyEvent()
+    onAccept()
+  }
+
+  const _handleReadMore = async ({showPartners = true}) => {
+    setShowPartners(showPartners)
+    setShowModal(true)
+    setShowNotification(true)
+    onConfigure()
+  }
+
+  /**
+   * This handler is used everytime the user clicks on something in the web
+   * in order to assume he's accepting our CMP if he's not trying to configure it and
+   * he's just navigating the website
+   */
+  const _handleClickOnDocument = ({target}) => {
+    const isClickableElement =
+      target.closest('a') || target.closest('button') || target.closest('input')
```

Comment:
Internet Explorer 11 has no support for Element.closest(). A polyfill is available though: 

https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill

Reply:
PR for it: https://github.com/SUI-Components/sui/pull/644